code/game/objects/items/weapons/boomerang.dm

```diff
@@ -1,25 +1,242 @@
+
+//////////////////////////////BOOMERANG ITEM////////////////////////////////
+
 /obj/item/weapon/boomerang
 	name = "boomerang"
 	desc = "A heavy, curved piece of wood used by Space Australians for hunting, sport, entertainment, cooking, religious rituals and warfare. When thrown, it will either deal a devastating blow to somebody's head, or return back to the thrower." //also used for shitposting
+	icon = 'icons/obj/boomerang.dmi'
 	icon_state = "boomerang"
+	item_state = null
+	inhand_states = list("left_hand" = 'icons/mob/in-hand/left/boomerang.dmi', "right_hand" = 'icons/mob/in-hand/right/boomerang.dmi')
 
 	w_class = W_CLASS_MEDIUM
-
-	throwforce = 16
+	force = 7
+	throwforce = 10
 	throw_range = 7
 	throw_speed = 5
 
-	force = 7
-
 	starting_materials = list(MAT_WOOD = CC_PER_SHEET_WOOD * 6)
 	w_type = RECYK_WOOD
 	flammable = TRUE
 
-/obj/item/weapon/boomerang/Destroy()
+	var/mob/originator = null
+	var/thrown = FALSE
+	var/throw_mult = 1
+	var/stun = 2
+	var/weaken = 3
```

Comment:
lol. lmao

Reply:
meanwhile tasers (which can't be grabbed or dodged by lying down) be like 
![image](https://github.com/user-attachments/assets/dd8753a2-05c4-4ae1-a263-4d5d9fcd65b5)

Reply:
would making thrown chewing gum deal 120 toxin damage(blocked if the targer is wearing a fake mustache or has a prime number of credits in their virtual wallet) be reasonable too?
jesus fuck man, a taser cant be made from 6 wooden planks